build-deploy.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+# build-deploy.sh - Wrapper script that runs build.sh and deploy.sh in sequence
+# This script maintains backward compatibility with the original build-deploy.sh
+
+set -e # Exit immediately if a command exits with a non-zero status
+
+# Function to print section headers
+print_header() {
+    echo "======================================================"
+    echo "🚀 $1"
+    echo "======================================================"
+}
+
+print_header "BUILD AND DEPLOY WRAPPER"
+echo "This script will run build.sh and deploy.sh in sequence."
+echo "You can also run them separately:"
+echo "  ./build.sh [prod|staging] [version_tag]"
+echo "  ./deploy.sh [prod|staging] [eu|nbg1|staging|masters] [version_tag] [subdomain] [--enable_basic_auth]"
+echo ""
+
+# Check command line arguments
+if [ $# -lt 2 ] || [ $# -gt 4 ]; then
+    echo "Error: Please specify environment and host, with optional subdomain"
+    echo "Usage: $0 [prod|staging] [eu|nbg1|staging|masters] [subdomain] [--enable_basic_auth]"
+    exit 1
+fi
+
+# Validate first argument (environment)
+if [ "$1" != "prod" ] && [ "$1" != "staging" ]; then
+    echo "Error: First argument must be either 'prod' or 'staging'"
+    echo "Usage: $0 [prod|staging] [eu|nbg1|staging|masters] [subdomain] [--enable_basic_auth]"
+    exit 1
+fi
+
+# Validate second argument (host)
+if [ "$2" != "eu" ] && [ "$2" != "nbg1" ] && [ "$2" != "staging" ] && [ "$2" != "masters" ]; then
+    echo "Error: Second argument must be either 'eu', 'nbg1', 'staging', or 'masters'"
+    echo "Usage: $0 [prod|staging] [eu|nbg1|staging|masters] [subdomain] [--enable_basic_auth]"
+    exit 1
+fi
+
+# Generate version tag
+VERSION_TAG=$(date +"%Y%m%d-%H%M%S")
+echo "Generated version tag: $VERSION_TAG"
+
+# Extract arguments
+ENV=$1
+HOST=$2
```

Comment:
I think you need to put quotes around these two to prevent code execution if the user passes a space in the argument.
```suggestion
ENV="$1"
HOST="$2"
```